tools/iree-opt-main.cc

```diff
@@ -9,5 +9,11 @@
 // Based on mlir-opt but registers the passes and dialects we care about.
 
 #include "iree/compiler/API2/ToolEntryPoints.h"
+#include "llvm/Support/PrettyStackTrace.h"
 
-int main(int argc, char **argv) { return ireeOptRunMain(argc, argv); }
+int main(int argc, char **argv) {
+  llvm::setBugReportMsg(
```

Comment:
this calls the method in IREEOptToolEntryPoint.cpp so it shouldn't be needed here